src/content/docs/en/reference/api-reference.mdx

```diff
@@ -2302,6 +2302,39 @@ The `code` property accepts human-readable versions of all HTTP status codes. Th
 
 The `message` property accepts a string. (e.g. "User must be logged in.")
 
+## Environment variables (`astro:env`)
+
+<p>
+<Since v="5.0.0" />
+</p>
+
+The `astro:env` API lets you configure a type-safe schema for environment variables you have set. This allows you to indicate whether they should be available on the server or the client, and define their data type and additional properties. For examples and usage instructions, [see the `astro:env` guide](/en/guides/environment-variables/#type-safe-environment-variables).
+
+### `getSecret()`
+
+<p>
+<Since v="5.0.0" />
+</p>
+
+The `getSecret()` helper function can be used for several things:
+
+1. **Get the raw value of a secret defined in your schema**: if you define a variable `FOO` of type boolean, using `getSecret()` will allow you to get the value as string:
+
+   ```js
+   import {
+      FOO, // boolean
+      getSecret
+   } from 'astro:env/server'
+
+   getSecret('FOO') // string | undefined
+   ```
+
+2. **Retrieve secrets not specified in your schema**: that can be useful if you need to get a secret depending on dynamic data, eg. a database or API.
+
+   :::note
+   If you need to use environment variables, we recommend you use `getSecret()` instead of `process.env` (or equivalent) because its implementation is provided by your adapter: It means you won't need to update all your calls if you switch adapters. It defaults to `process.env` in dev and build.
+   :::
```

Comment:
```suggestion
If you need to retrieve environment variables programmatically, we recommend you use `getSecret()` instead of `process.env` (or equivalent) because its implementation is provided by your adapter: It means you won't need to update all your calls if you switch adapters. It defaults to `process.env` in dev and build.
```